crates/rpc/rpc/src/debug.rs

```diff
@@ -687,7 +687,7 @@ where
             .into_iter()
             .map(|receipt| {
                 let mut buf = Vec::new();
-                receipt.with_bloom().encode(&mut buf);
+                receipt.with_bloom().encode_inner(&mut buf, false);
```

Comment:
I believe this is is also `enveloped` encoded

same as:

https://github.com/paradigmxyz/reth/blob/1156be213d9f5f87941e41067cc8168bf545c3a5/crates/primitives/src/transaction/mod.rs#L1110-L1128

so let's also add the functions + docs to receipt type